### PR TITLE
refactor: add dinero.js, change price schema

### PR DESCRIPTION
**Pull request by Keitarofreeman**
This pull request is relate to the issue: [https://github.com/tabetalt/backoffice/issues/99](url)

re #99

### CODE REVIEW

File: src/modules/catalog/product/ProductUpdate.tsx [hunk 1]
```diff
@@ -54,7 +55,13 @@ const ProductUpdate: React.FC = () => {
       ]) as ProductUpdateInput;
 
       if (values.price) {
-        input.price = values.price.replace(',', '.') * 100;
+        input.price = {
```
Comment (cobraz) [inline src/modules/catalog/product/ProductUpdate.tsx:58]: Should probably be a parser that can be tested?

File: src/helpers/index.ts [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
-export const formatPrice = (
-  formattedPrice?: FormattedPrice | null,
-  currency: string | null = 'NOK'
-): string => {
-  const amount = numeral(
-    formattedPrice ? formattedPrice.netAmount / 100 : 0
-  ).format('0.00');
-  return currency ? `${amount} ${currency}` : amount;
+export const formatPrice = (formattedPrice?: Price | null): string => {
```
Comment (cobraz) [inline src/helpers/index.ts:4]: Maybe add some tests on this?

**Review by cobraz — COMMENTED** [review]
Just some nitpicking :)

File: src/helpers/index.ts (hunk 2 above)
Comment (cobraz) [reply src/helpers/index.ts:4]: Something like this maybe?

```typescript
it('should format price', () => {
	const f = formatPrice({ ... });
	expect(f).toMatchInlineSnapshot();
})
``` 

What do you think?

File: src/modules/catalog/product/ProductUpdate.tsx (hunk 1 above)
Comment (Keitarofreeman) [reply src/modules/catalog/product/ProductUpdate.tsx:59]: We added tests in update for this PR.

File: src/helpers/index.ts (hunk 2 above)
Comment (Keitarofreeman) [reply src/helpers/index.ts]: We added tests in update for this PR.

**Pull request merged**